deployment/k8s-config/kustomize/base/skaha/config/add-user.yaml

```diff
@@ -0,0 +1,55 @@
+# Production version of add-user.
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: "{skaha.adduser.jobname}"
+spec:
+  ttlSecondsAfterFinished: 3600
+  backoffLimit: 0
+  template:
+    spec:
+      restartPolicy: Never
+      containers:
+      - name: "skaha-add-user"
+        image: images.canfar.net/skaha-system/add-user:1.2
+        imagePullPolicy: Always
+        command: ["/usr/bin/add-user"]
+        args: ["{skaha.userid}", "{skaha.userquotagb}"]
```

Comment:
The params to be replaced in this file should use the same syntax as the others, ie "${param}".  So, it's just missing the $.